docs/security/hadoop_credentials_manager.md

```diff
@@ -83,8 +83,8 @@ The necessary configurations for hdfs and hive to obtain delegation token are as
 
 Key | Meaning | value
 --- | --- | ---
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hadoop.security.authentication</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>Set the authentication for the cluster</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>kerberos</div>
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.uris</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>URI for client to contact metastore server</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>thrift://{metastoreHost}:{metastorePort}}</div>
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.sasl.enabled</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>If true, the metastore thrift interface will be secured with SASL.Clients must authenticate with Kerberos.</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>true</div>
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.kerberos.principal</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>The service principal for the metastore thrift server. The special string _HOST will be replaced automatically with the correct host name.</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>for example hive/_HOST@${realm}</div>
-<div style='width: 40pt;word-wrap: break-word;white-space: normal'>hive.metastore.kerberos.keytab.file</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>The path to the Kerberos Keytab file containing the metastore thrift server's service principal.</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>for example /etc/security/keytabs/hive.service.keytab</div>
\ No newline at end of file
+hadoop\.security<br>\.authentication|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>Set the authentication for the cluster</div>|<div style='width: 40pt;word-wrap: break-word;white-space: normal'>kerberos</div>
```

Comment:
let's quote it by backquote "\`" `hadoop\.security<br>\.authentication` or `<code>hadoop.security.authentication</code>`

Reply:
See more information in previous discussion https://github.com/apache/incubator-kyuubi/pull/1890

Reply:
> 

It seems that the previous part of this document is also not modified according to #1890

Reply:
> If possible, when you touch the document, would you please build the document in local and paste the previous screenshot in your PR description?

Ok ,I will paste

Reply:
> It seems that the previous part of this document is also not modified according to #1890

Let's fix it as much as possible

Reply:
> It seems that the previous part of this document is also not modified according to #1890

Sorry I didn't notice this document when I changed it.
There are two more.
`authentication.md`
`kinit.md`